.github/workflows/compatibility.yml

```diff
@@ -11,12 +11,12 @@ jobs:
       fail-fast:    false
       max-parallel: 10
       matrix:
-        woocommerce: [ '4.4.0', '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0' ]
+        woocommerce: [ '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0-rc.1']
         wordpress:   [ 'latest' ]
         php:         [ '7.4' ]
         include:
           # Edge case: oldest dependencies compatibility
-          - woocommerce: '4.4.0'
+          - woocommerce: '4.5.0'
             wordpress:   '5.4'
```

Comment:
I think it's still good to update this WP version from 5.4 to 5.6. That's because, in practice, WCPay is requiring WP version >= 5.6.

Reply:
But the E2E tests are failing. I think we need to fix that first then.

Reply:
> I think we need to fix that first then ( on a separate PR ).

👍 Agreed!

Reply:
Let me spin up another PR as I had some ideas about it in this test PR https://github.com/Automattic/woocommerce-payments/pull/3346

Reply:
> Let me spin up another PR as I had some ideas about it in this test PR #3346

Here is the PR https://github.com/Automattic/woocommerce-payments/pull/3597